fcl/standardMessageDestinations.fcl

```diff
@@ -138,6 +138,8 @@ mf_coutInfo:
    Configuration : { limit : 0 }
    HITS : { limit : 0 }
    COSMIC_STEPPOINTS : { limit : 0 }
+   Summary : { limit : 0 }
+   INFO : { limit : 0 }
  } # end categories
```

Comment:
Does limit:0 disable the printing?

Reply:
> Does limit:0 disable the printing?

Yes, the number is how many prints of this message are allowed,
so zero means no printing

Reply:
I believe that if you ask for the message facility end of job summary report you get the number of messages in each category, but only for categories with > 0 messages.  So there is value in setting a limit of 0 as a default.

Ray: do I have that right?  Do we have that summary on by default?

Reply:
Hi Ray,

> Yes, the number is how many prints of this message are allowed, so zero means no printing

What is not clear to me is the role of all those INFO printouts to message facility in the code.

I agree that we want to minimize routing "everything is OK" printouts in production jobs, but there is always a tension between production and code development.   If a new developer starts working on  an Offline module, ideally things would be setup for all the info printouts to show up in their tests, I think.   But the same printouts would be off for production running.

When I first started working on Offline,  I realized that not all my prints to message_facility showed up (I do not remember the details).   I guessed there were some obscure settings somewhere to make them work, but that complexity immediately disqualified the use of message facility for code development/debugging in my mind.   I just started printing  all the stuff I needed to see to std::cout.    I am not saying that this is the right thing to do, but this is how it worked in practice.

I think the "out-of-the-box" setting  of Offline should be for production running.   If we can have a simple instruction for code developers, "after git clone and muse setup do this to enable INFO printouts in all your interactive jobs",  then having all those message facility prints in our C++ code makes sense.    Otherwise they are basically dead code, not used in production, not used for code development.   May be just for experts to localize a problem without re-compiling the code.

I think addressing this question is outside of the scope of the current PR, which I am approving, but we should keep thinking about this.
Andrei

Reply:
I agree that this is out of scope.  I have relaunched the tests and merged.  I will create an issue to follow up.